dockstring/utils.py

```diff
@@ -63,6 +63,13 @@ def get_vina_filename() -> str:
     system_name = platform.system()
     if system_name == 'Linux':
         return 'vina_linux'
+    if system_name == 'Darwin':
+        warnings.warn(
+            "Although Mac use is supported, docking scores on Mac generally *do not* "
```

Comment:
`"Although Mac use is supported, docking scores on Mac do not always perfectly match scores from Linux. Therefore, extra care should be taken when comparing results to other platforms. In particular, the baselines in the DOCKSTRING paper were computed on Linux, so please do not directly compare your docking scores to the scores reported on the paper."